### PR TITLE
Add sautéed kale and white bean soup recipes

### PR DESCRIPTION
**Pull request by diego-codes**

### CODE REVIEW

File: soup/white-bean-brothy-soup.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# White bean brothy soup
+
+This recipe is a "European" tasting soup that is bright and brothy. It is time-consuming, but the labor is minimal. This recipe is based on Bon Appetit's [video recipe](https://www.youtube.com/watch?v=PXqoyt4vsPA).
+
+**Note:** Make sure that you plan for this recipe to have enough time to soak beans.
+
```
Comment (NicMoe) [inline soup/white-bean-brothy-soup.md]: add space

File: sides/sauteed-kale.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+A quick and tasty way to cook up a bunch of kale. 
+
+
+## Ingredients
+
+- 2 - 3 tbsp olive oil
+- 1 - 2 garlic cloves thinly sliced
+- 1/2 tsp red pepper chili flakes
+- 1/3 of a bag of chopped kale (enough to cover the entire pan)
```
Comment (NicMoe) [inline sides/sauteed-kale.md]: how big is the bag?

File: soup/white-bean-brothy-soup.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# White bean brothy soup
```
Comment (NicMoe) [inline soup/white-bean-brothy-soup.md]: I feel like this recipe should be called "brothy white bean soup" instead

**Review by NicMoe — CHANGES_REQUESTED** [review]
Overall looks good. Check out my questions / comments.

**Comment by diego-codes** [comment]
Thanks, @NicMoe! This is ready to review again.

**Review by NicMoe — APPROVED** [review]
Looks good!

**Pull request merged**